definitions/tools/custom_merge_sv_records.cwl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "merges nearby DEL/DUP records within a certain window distance"
+
+baseCommand: ["/bin/bash", "run_merge.sh"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "apaul7/analysis:1.0.0"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: "run_merge.sh"
+        entry: |
+          #!/bin/bash
+          set -eou pipefail
+          INPUT="$1"
+          OUTPUT="$2"
+          DISTANCE="$3"
+          /usr/local/bin/python3 /opt/git/merge-sv-records/merge.py -i $INPUT -o $OUTPUT -w $DISTANCE
+
+          /usr/local/bin/bgzip $OUTPUT
+          /usr/local/bin/tabix -p vcf $OUTPUT.gz
```

Comment:
Maybe we should quote the parameters with variables on these lines just in case the names have some weird characters in them.

Reply:
I changed to a python script for this tool